encrypted-media-respec.html

```diff
@@ -2487,6 +2487,9 @@
           Specifically, the CDM SHOULD NOT store session data during the <a def-id="generateRequest"></a> algorithm.
           This ensures that the application is aware of the session and knows it needs to eventually remove it.
         </p>
+        <p><em>All</em> data associated with a session MUST be cleared when the session is cleared, such as when a <a def-id="remove"></a> request is completed in the <a def-id="update"></a> algorithm. 
+          See <a href="#persistent-state-requirements">Persistent Data</a>.
+        </p>
```

Comment:
I'm fine with that approach. I want to avoid limiting this language to the current session types and algorithms to avoid rot. Thus, I would like to keep a general statement and example. I've made minor modifications to your suggestion below.

> *All* data associated with a session MUST be cleared when the persisted session is cleared, such as in `update` when the processing of an acknowledgement of the _record of key usage_ or _record of license destruction_.